src/components/modals/FormStepSaveModal.js

```diff
@@ -131,7 +138,8 @@ const FormStepSaveModal = ({isOpen, closeModal, onSaveConfirm, suspendFormUrl, s
         <Body modifiers={['big']}>
           <FormattedMessage
             description="Form save modal body text"
-            defaultMessage="Enter your email address to get an email to resume the form at a later date"
+            defaultMessage="Enter your email address to get an email to resume the form at a later date. This can be done on any device where you open the link. De link remains valid for {numberOfDays} day(s)."
+            values={{numberOfDays: suspendFormUrlLifetime}}
```

Comment:
https://formatjs.io/docs/core-concepts/icu-syntax/#plural-format so you can manage the pluralization inside the message.